models/tmp/stg_greenhouse__tag_tmp.sql

```diff
@@ -1 +1 @@
-select * from {{ var('tag') }}
+select * from {{ var('tag') if does_table_exist('tag') else var('tags') }}
```

Comment:
```suggestion
select * from {{ var('tags') if var('greenhouse__using_tags', does_table_exist('tags')) else var('tag') }}
```
Unless there is a reason not to, I suggest to prioritize "tags" since that is the newer table in the rare chance both tables exist.

I also suggest adding a var like `greenhouse__using_tags` so the version can be manually set if necessary.

Reply:
In the scoping ticket, I think the preference was on 'tag' and 'user' to prioritize the old names, but I'll let @fivetran-joemarkiewicz  weigh in.

I can make those `greenhouse__using_tags` and `greenhouse__using_users` variables, but ideally couldn't a user leverage the identifier variables to specify which version to use, given they have both?

Reply:
@fivetran-reneeli The identifiers are unique to each source spelling, so they can't act as a switch. This only matters if both tables exist and a user wants to change from the default behavior.

@fivetran-joemarkiewicz My thought on prioritizing the new name is so the data would be current in the case both names exist.

Reply:
Following what we discussed live, I flipped the order to look for the plural version first, and noted it in the CHANGELOG.